src/ResourceManager/Insights/Commands.Insights/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.Monitor" version="0.16.1-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Monitor" version="0.19.1-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
```

Comment:
@pkamath13 @gucalder you can remove this reference to `Microsoft.Azure.KeyVault.Core` in this file

Reply:
@cormacpayne Any change needed in the wxi file?  Is that automatically generated?